backend/services/implementations/schoolService.ts

```diff
@@ -117,13 +100,13 @@ class SchoolService implements ISchoolService {
     }
 
     return {
-      id: newSchool.id,
+      id: newSchool.id.toString(),
```

Comment:
why do we have to do this type cast here? seems like we don't need it in other services?